x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -24,6 +24,8 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
 
     const detailPanel: CSSObject = {
       height: `${height}px`,
+      borderLeft: `${euiTheme.border.thin} !important`,
```

Comment:
Are all these !important's needed? Unless there is a competing css style that can't be solved with more specificity, I'd try to avoid it's use. https://www.reddit.com/r/css/comments/d4kf7b/why_is_it_considered_not_good_to_use_important/